.github/workflows/hide_bot_comments.yml

```diff
@@ -0,0 +1,24 @@
+name: Hide Bot Comments
+on:
+  push:
+    branches: [master]
+  pull_request:
+    types: [opened, synchronize, reopened]
+    branches: [master]
```

Comment:
It seems a little aggressive to be hide all comments when a push to master occurs 🤔 

~Maybe only hide them when a new comment is posted?~ 
↪️ It runs for every pull request which is not better.

https://docs.github.com/en/actions/reference/events-that-trigger-workflows#issue_comment

You can filter by pull request and byt actor (ie only run when the bot posts a comment)

```suggestion
  issue_comment:
    types: [created]
```

Reply:
this is not yet decided. I may try to implement it in the pipeline instead. you know there are several factors to consider:
- I don't want to fail/block PRs if this action fails for whatever reason
- it shouldn't make builds longer
- we don't want to waste our free GitHub minutes (I guess there is a limit)
- more network requests you make - more chances of failures (e.g. unexpected errors)

so running on every PR or every comment might be too wasteful.
also scanning all PRs and all comments also might be wasteful.

Reply:
> I don't want to fail/block PRs if this action fails for whatever reason

Pretty sure that is the case here and with the one I proposed too 🙊 

> more network requests you make - more chances of failures

I must say the availability with GitHub API is very good, in pending review I have only ever gotten 1 failure because their backend failed.

Actions run in their own network so it's much more stable than using the WWW

***

the action I mention https://github.com/conan-io/conan-center-index/pull/4937#issuecomment-801905566 only runs for that exact PR which is nice